modules/tensor_mechanics/test/tests/smeared_cracking/tests

```diff
@@ -8,10 +8,34 @@
     use_old_floor = true
     compiler = 'GCC CLANG'
   [../]
-  [./single_edge_notch]
+  [./cracking]
     type = 'Exodiff'
-    input = 'crack2d.i'
-    exodiff = 'crack2d_out.e'
-    rel_err = 1.0e-03
+    input = 'cracking.i'
+    exodiff = 'cracking_out.e'
+    rel_err = 1e-04
```

Comment:
This one seems to pass, but I'd say go with the sm settings, and use abs_zero instead of rel_err